src/Orleans/AssemblyLoader/AssemblyProcessor.cs

```diff
@@ -145,17 +145,24 @@ private static void ProcessAssembly(Assembly assembly)
             // Process each type in the assembly.
             foreach (TypeInfo type in assemblyTypes)
             {
-                string typeName = type.FullName;
-                if (Logger.IsVerbose3)
+                try
                 {
-                    Logger.Verbose3("Processing type {0}", typeName);
+                    string typeName = type.FullName;
+                    if (Logger.IsVerbose3)
+                    {
+                        Logger.Verbose3("Processing type {0}", typeName);
+                    }
+                    if (shouldProcessSerialization)
+                    {
+                        SerializationManager.FindSerializationInfo(type);
+                    }
+    
+                    GrainFactory.FindSupportClasses(type);
                 }
-                if (shouldProcessSerialization)
+                catch (Exception exception)
```

Comment:
I assume it was a conscious decision to now log an error but continue processing other types and start the silo, even after those errors occurred. Correct?